Week-1/Assignment 1.2/style.css

```diff
@@ -0,0 +1,6786 @@
+.first-section{
+    display: flex;
+    align-items: flex-start; 
+    
+}
+#img1, #img2, #img3 {
+    width: 30px;
+    margin-right: 2px;
+}
+
+h6#head2 {
+  position: relative;
+  top: -14px;
+}
+
+.playbox{
+    display: flex;
+    align-items: center;
+}
+header.header_section {
+    padding: 20px;
+    background-color: rgb(30, 29, 29);
+}
+
+.headline {
+  color: white;
+}
+
+i.las.la-play {
+  position: relative;
+  top: 5px;
+}
+
+i.las.la-plus {
+  position: relative;
+  top: 4px;
+}
+
+i.las.la-ellipsis-h {
+  position: relative;
+  top: 5px;
+}
+
+.imageplustext {
+  display: flex;
+  align-items: center;
+  position: relative;
+  top: 62px;
+}
+
+.footericon {
+  display: flex;
+  align-items: center;
+  position: relative;
+  right: -1220px;
+}
+
+i.las.la-pause {
+  position: relative;
+  right: -54px;
+}
+
+i.las.la-step-forward {
+  position: relative;
+  right: -108px;
+}
+
+i.las.la-undo-alt {
+  position: relative;
+  right: -162px;
+}
+
+i.las.la-random {
+  position: relative;
+  right: -216px;
+}
+
+.icon {
+    font-size: 15px;
+    line-height: 10px;
+    color: white;
+    margin-right: 20px;
+}
+
+.text_img {
+    margin-left: 43px;
+}
+table {
+    font-family: Lato;
+    border-collapse: collapse;
+    width: 96%;
+    margin-left: 10px;
+  }
+
+  td, th {
+    border: 1px solid #dddddd;
+    text-align: left;
+    padding: 8px;
+  }
+  tr:nth-child(even) {
+    background-color: #dddddd;
+  }
+  .last_section{
+    padding: 20px;
+    background-color: rgb(111, 111, 217);
+    display: flex;
+    
+  }
+  #footerimg {
+    width: 100px;
+    
+  }
+  .ftext {
+    margin-left: 20px;
+}
+
+body {font-family: "Lato", sans-serif;}
```

Comment:
can we please have an hierarchy for the css classes? Lets have the common ones to the top followed by how the HTML DOM structure is. This will be easy for us to read and maintain.

Reply:
Yes, Completed.